packages/url_launcher/url_launcher_platform_interface/test/url_launcher_platform_test.dart

```diff
@@ -0,0 +1,75 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:url_launcher/url_launcher.dart';
+import 'package:url_launcher_platform_interface/url_launcher_platform_interface.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+
+  // TODO(hterkelsen): Re-enable this test once we land the corresponding
+  //     change in package:url_launcher
+  group('UrlLauncher with mock platform', () {
```

Comment:
This is testing the app-facing plugin and not the platform interface. Not sure there's anything to test for `UrlLauncherPlatform`.

I think it will make sense for the tests in this package to be testing the platform channel implementation (with a mock method handler).

Reply:
Added a test for `MethodChannelUrlLauncher`